README.md

```diff
@@ -100,7 +112,9 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema is how are database will store its information, and what attributes those relationships it will have such
+as one-to-many or many-to-many. Using primary and foreign keys we would establish our primary in our Person, and then
+a foreign key to the wish to assign that wish to a person.
```

Comment:
This is good. The only thing I'd add is that the schema defines the column names and *data types* for each column in each table.